app/containers/InterventionPage/ShareFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useMemo } from 'react';
+import { Col } from 'react-grid-system';
+
+import globalMessages from 'global/i18n/globalMessages';
+
+import Select from 'components/Select';
+import { IntlShape } from 'react-intl';
+import { SelectOption } from 'components/Select/types';
+
+type Props = {
+  formatMessage: IntlShape['formatMessage'];
+  onChange: (values: string) => void;
+  active: string;
+};
+
+const ShareFilter = ({ formatMessage, onChange, active }: Props) => {
+  const options = useMemo(
+    () =>
+      Object.keys(globalMessages.sharingFilters).map((status) => ({
+        value: status,
+        // @ts-ignore
+        label: formatMessage(globalMessages.sharingFilters[status]),
+      })),
+    [globalMessages.statuses],
+  );
+
+  const selectValue = active
+    ? {
+        value: active,
+        // @ts-ignore
+        label: formatMessage(globalMessages.sharingFilters[active]),
+      }
+    : null;
+
+  const handleChange = (selectedValue: SelectOption<string>) =>
+    onChange(selectedValue.value);
+
+  return (
+    <Col>
+      <Select
+        width={400}
```

Comment:
Remove `width` and define select's width by grid column size

before

https://github.com/Michigan-State-University/cias-web/assets/86963976/752ce2ae-7bff-49c4-bc64-7e9f3bfea5ee

after

https://github.com/Michigan-State-University/cias-web/assets/86963976/dcbe7339-cdbf-4ddd-b660-af491cd1ca70